kubernetes/resources_api/base/deployment.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: resources-api
+spec:
+  replicas: 2
+  revisionHistoryLimit: 5
+  template:
+    spec:
+      containers:
+      - name: app
+        image: #operationcode/operationcode_backend:latest
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 3000
+        env:
+        - name: FLASK_APP
```

Comment:
Just out of curiosity, why is this an env var?

Reply:
>The flask command is installed by Flask, not your application; it must be told where to find your application in order to use it. The FLASK_APP environment variable is used to specify how to load the application.
http://flask.pocoo.org/docs/1.0/cli/
You run the API by calling `docker-compose run -p 8000:8000 resources-api flask run -p 8000 -h 0.0.0.0` so `flask run` needs that environment variable to find the app